clip/image_config.py

```diff
@@ -0,0 +1,28 @@
+CLIP_IMAGE_CONFIG = {'openai/clip-vit-base-patch32': {
```

Comment:
We want to pull these from HuggingFace instead of hardcoding constants. This solves the problem, but hurts maintenance.